### PR TITLE
Add ts insert column

### PR DESCRIPTION
**Pull request by vianneyPL**
Add insert columns.
Add ColumnInfos to get the timeseries current information.

### CODE REVIEW

File: entry_timeseries.go [hunk 1]
```diff
@@ -83,6 +91,33 @@ func (entry TimeseriesEntry) Create() error {
 	return makeErrorOrNil(err)
 }
 
+// InsertColumns : insert columns in a existing timeseries
+func (entry *TimeseriesEntry) InsertColumns(cols ...TsColumnInfo) error {
+	colsToInsert := TsColumnInfos(cols)
```
Comment (mkurdej) [inline entry_timeseries.go]: It may be purely personal, but I find `TsColumnInfos` (with 's') more confusing than clear.
I'll see the use of `[]TsColumnInfo` (without 's') clearer to the reader.

File: entry_timeseries.go [hunk 2]
```diff
@@ -83,6 +91,33 @@ func (entry TimeseriesEntry) Create() error {
...
+	alias := C.CString(entry.alias)
+	columnsCount := C.qdb_size_t(len(colsToInsert))
+	columnsArray := colsToInsert.toStructC()
+	var columns *C.qdb_ts_column_info_t
+	if columnsCount != 0 {
```
Comment (mkurdej) [inline entry_timeseries.go]: Why do you check for that? You may leave the C API handle the validation of user input, hmm?

File: entry_timeseries_test.go [hunk 3]
```diff
@@ -72,6 +72,27 @@ var _ = Describe("Tests", func() {
 				err := timeseries.Create()
 				Expect(err).ToNot(HaveOccurred())
 			})
+			Context("Insert Columns", func() {
```
Comment (mkurdej) [inline entry_timeseries_test.go]: ⚠️ You should add a test that `Create`s a timeseries with zero columns.
